plugin/core/sessions.py

```diff
@@ -254,14 +254,14 @@ def get_initialize_params(variables: Dict[str, str], workspace_folders: List[Wor
         },
         "signatureHelp": {
             "dynamicRegistration": True,
+            "activeParameterSupport": True,
```

Comment:
It's not in the correct object now (also not sorted :)).

Reply:
Oops. BTW they are not sorted strictly alphabetically in the other capabilities too. DynamicRegistration seems to be the first entry usually, even if there is another key starting with lower alphabetical letters. Initially I used the order how it is defined in the specs.

Reply:
Such arbitrary logic just creates confusion and unnecessary review noise like here IMO. But it's not worth spending more time on it :)

Reply:
I mean you was it who suggested to order alphabetically rather than follow the order from the specs ;)

Reply:
Yeah. And i wished it would be like that but it's not because the dynamic one is arbitrary :)
It's not your fault though 😉